lib/lorekeeper/simple_logger.rb

```diff
@@ -37,16 +37,21 @@ def inspect
       end
     end
 
-    def exception(exception, custom_message = nil, custom_data = nil)
+    def exception(exception, custom_message = nil, custom_data = nil, level = :error)
+      log_level = METHOD_SEVERITY_MAP[level]
+      unless log_level
+        return log_data(METHOD_SEVERITY_MAP[:warn], "Logger exception called with an invalid level: '#{level}'.")
+      end
+
       if exception.is_a?(Exception)
         backtrace = exception.backtrace || []
         message = custom_message || exception.message
-        error_with_data("#{exception.class}: #{message}", backtrace.join("\n"))
+        log_data(log_level, "#{exception.class}: #{message}, data: #{backtrace.join("\n")}")
```

Comment:
behavior change, but I always regretted to not add `exception.message` here somewhere. It comes in handy. As this typically is just logging on the terminal, we are quite free to change this.

Reply:
ah, should I revert that back to using `error` then?

Reply:
is good. I'm saying, instead of 
`log_data(log_level, "#{exception.class}: #{message}, data: #{backtrace.join("\n")}")`
do